scripts/test-fuzzed-MAC.py

```diff
@@ -21,6 +21,9 @@ def main():
     """check if incorrect MAC hash is rejected by server"""
     conversations = {}
 
+    #aead = True # GCM
+    aead = False # CBC
```

Comment:
I would prefer if that was an additional loop, external to the `for pos, val`

or simply a separate test script